planner/core/logical_plans.go

```diff
@@ -1008,12 +1008,21 @@ func (lt *LogicalTopN) isLimit() bool {
 // LogicalLimit represents offset and limit plan.
 type LogicalLimit struct {
 	baseLogicalPlan
+	schema *expression.Schema
```

Comment:
In my memory, `LogicalLimit.ResolveIndices` and `LogicalLimit.BuildKeyInfo` should be implemented when Schema is added into a logical plan and if not, there should be some tests to break.
cc @winoros , it that true?

Reply:
I find that `ResolveIndices` is used to correct the column index of physical plan member(s), e.g., [`ByItems`](https://github.com/pingcap/tidb/blob/c704b9756e20d172da32e95f2c7bec3c40dbaff3/planner/core/resolve_indices.go#L417) of `PhysicalSort`.

So `Limit` should not need to implement this method.

Reply:
Implementation of [`LogicalLimit.BuildKeyInfo`](https://github.com/pingyu/tidb/blob/de436ef0107a2664e57f529834eb1717f9e51982/planner/core/rule_build_key_info.go#L97) was fixed. PTAL~